app/models/planning_application.rb

```diff
@@ -223,7 +223,7 @@ def has_validation_date?
   end
 
   def public_comment_present
-    if decision == "refused" && public_comment.blank?
+    if decision && public_comment.blank?
```

Comment:
`decision.present?` would be more defensive here. While the application currently doesn't ever create a decision of a blank string, it's not impossible and we might do that in the future, and if we did I'd assume that we'd want a blank string to mean "no decision".